tests/cdkproxy/test_pipeline_stack.py

```diff
@@ -28,6 +28,32 @@ def patch_methods(mocker, db, pipeline2, env, pip_envs, org):
         'dataall.cdkproxy.stacks.pipeline.PipelineStack.get_pipeline_environments',
         return_value=pip_envs,
     )
+    mocker.patch(
+        'dataall.cdkproxy.stacks.pipeline.PipelineStack._set_env_vars',
+        return_value= ({
+            'AWS_REGION': env.region,
+            'AWS_DEFAULT_REGION': env.region,
+            'CURRENT_AWS_ACCOUNT': env.AwsAccountId,
+            'envname': 'pytest',
+        }, True)
+    )
+    mocker.patch(
+        'dataall.cdkproxy.stacks.pipeline.PipelineStack._set_env_vars',
+        return_value=({
+                          'AWS_REGION': env.region,
+                          'AWS_DEFAULT_REGION': env.region,
+                          'CURRENT_AWS_ACCOUNT': env.AwsAccountId,
+                          'envname': 'pytest',
+                      }, True)
```

Comment:
Need to return `None` rather than `True` for `_check_repository`... otherwise it will assume CodeCommit Repo exists and not create one causing an assertion error